low-level/files/opt/seagate/sspl/setup/sspl_post_install.py

```diff
@@ -0,0 +1,153 @@
+#!/bin/env python3
+
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+
+###################################################################
+# post_install script configures ryslog, sspl service and required
+# site packages for SSPL. This is executed from sspl_setup script.
+###################################################################
+
+import os
+import errno
+import shutil
+import distutils.dir_util
+
+# using cortx package
+from cortx.sspl.bin.error import SetupError
+from cortx.sspl.bin.sspl_constants import (REPLACEMENT_NODE_ENV_VAR_FILE,
```

Comment:
why do we have error and sspl_constants modules inside bin package? seems odd.

Reply:
agree. @thavanathan suggest dir structure to maintain these files in different dir other than bin

Also sspl_contants & setup_error (rename) are not cli, so let that be WITH py extension. Same applies to other files under bin